### ISSUE TITLE
Documentation / model link error: RetinaFace and SCRFD identical 

### ISSUE THREAD

**Issue by julie-sony** [deepinsight/insightface#2197]
Hi there, 

It seems as though the "RetinaFace" models referenced, and linked to, here: 
https://github.com/deepinsight/insightface/tree/master/model_zoo

...Actually produce identical results to the SCRFD models linked here. You also report identical fairness analysis results on them. 
https://github.com/deepinsight/insightface/tree/master/python-package

I assume "RetinaFace" in the former README should really be "SCRFD"; in this case, can you please provide .onnx files for RetinaFace? Thanks!

**Comment by nttstar** [comment]
Yes, they're actually identical in python package. There are no onnx files for RetinaFace.
